tests/testthat/test-rename.r

```diff
@@ -1,4 +1,7 @@
-context("Rename")
+#################################################
```

Comment:
Sorry, was in C++ mode. Yes - the standard for section headings is 

`# Section -----------------------------------`